specifics/Dhyana/Dhyana.cpp

```diff
@@ -358,9 +262,6 @@ void Dhyana::get_device_property()
 	/*----- PROTECTED REGION ID(Dhyana::get_device_property_after) ENABLED START -----*/
 	
 	//	Check device property data members init
-	/*PropertyHelper::create_property_if_empty(this, dev_prop,"15","TemperatureTargetAtInit");	
```

Comment:
Attention, il faut remettre la ligne:
`PropertyHelper::create_property_if_empty(this, dev_prop,"15","TemperatureTargetAtInit");`

ok pour les 2 autres